metabase/metabase
# Add "basic" label to basic actions

Epic https://github.com/metabase/metabase/issues/28534

Please review this PR by each commit:
1. Add `describe.each` to allow running tests from IDE. No test changes
2. Add basic label + tests.

How to verify:
- Enable actions for you database in Admin -> Database -> Select DB -> Enable model actions
- Go to any model -> Info sidebar -> Model details -> Actions
- Create basic actions if you don't have them already
- Make sure they have "Basic" label in the subtitle

<img width="1043" alt="Screenshot 2023-04-07 at 18 37 07" src="https://user-images.githubusercontent.com/8542534/230636513-a57e9e2e-91d9-44cb-ab49-9a2a2e235931.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29892)
